src/render/SDL_yuv_sw.c

```diff
@@ -26,6 +26,48 @@
 
 #include "SDL_yuv_sw_c.h"
 
+/*
+ * Output 'pitch' that can be used with SDL_ConvertPixels()
+ */
+int SDL_SW_SizeYUV(Uint32 format, int w, int h, int *pitch)
+{
+    const int sz_plane = w * h;
```

Comment:
It might be out of scope for your change, but these values need the same overflow protection that we use when creating RGB surfaces, using `SDL_size_mul_overflow()` and checking the return values. Maybe this function should return a result code and fill in both size and pitch pointers passed in?

Reply:
I've change the return code and use size_t for the size.
but no real overflow check yet